src/frontend/src/components/server-icon/index.tsx

```diff
@@ -23,6 +32,11 @@ function ServerIcon({ imageUrl, imageSize = 48, label, isActive, hasAlarm, ...pr
       type='button'
       className='flex items-center w-full justify-center relative'
       {...props}>
+      {unreadCount !== undefined && unreadCount > 0 && (
```

Comment:
다음과 같이 한줄로 바꿔도 될 것 같아요!
```suggestion
      {Number(unreadCount) > 0 && (
```

Reply:
좋습니다 ! 반영해둘게요